metadeploy/api/salesforce.py

```diff
@@ -177,9 +177,7 @@ def _get_access_token(*, org_result, scratch_org_config):
         SF_CLIENT_ID, SF_CLIENT_SECRET, SF_CALLBACK_URL, scratch_org_config.instance_url
     )
     auth_result = oauth.get_token(org_result["AuthCode"]).json()
-    # TODO: This needs to be updated to `_sfdx_info` instead of `_scratch_info` when
-    # MetaDeploy gets updated to cumulusci 3.20.1
-    scratch_org_config.config["access_token"] = scratch_org_config._scratch_info[
+    scratch_org_config.config["access_token"] = scratch_org_config._sfdx_info[
```

Comment:
@davisagli I'm not 100% on these 3 changes, but they seem to work functionally 🤷

Reply:
@jgerigmeyer yeah, this is right for cumulusci >= 3.20.0. The internal structure is used for the same thing but got renamed.